stable/v3io-configs/templates/_script.tpl

```diff
@@ -114,3 +114,22 @@ set -e
 {{ include "v3io-configs.script.httpHealthCheck" . }}
 {{ include "v3io-configs.script.javaHealthCheckTest" . }}
 {{- end -}}
+
+{{- define "v3io-configs.script.v3ioDaemonHealthCheck" -}}
+CONFIGURED_V3IOD=$(grep "socket.host" $IGZ_DATA_CONFIG_FILE | cut -f2 -d"=")
+if [ "${CONFIGURED_V3IOD}" != "CURRENT_NODE_IP" ]; then
+  run_id_file=/tmp/v3iod_run_id
+  run_id=`/var/run/iguazio/daemon_health/healthz --host ${CONFIGURED_V3IOD}`
+  if [ -n "$run_id" ]; then
```

Comment:
if i understand the healthz code, in happy_flow this run_id bash var will have a value like "response status: 130192310" because we are capturing the whole response line (which is somewhat misleading).
1. Now, is it possible that in case of timeout waiting to the udp response that run_id will capture the "Failed to read from UDP.." response and will be "happy" with it (not declare liveness issue?)
2. second quetsion what happens in case of timeout waiting for the udp response? will run_id have a value not equal to null - which cause us to skip the check?

Reply:
fixed the script. 
if the response is empty or not match the wanted prefix exit